pyrate/core/config.py

```diff
@@ -141,6 +141,17 @@
 APS_ELEVATION_MAP = 'elevationmap'
 APS_ELEVATION_EXT = 'APS_ELEVATION_EXT'
 
+
+# phase closure
+PHASE_CLOSURE = 'phase_closure'
+LARGE_DEV_THR = 'large_dev_thr'
+THRESHOLD_TO_REMOVE_IFG = 'threshold_to_remove_ifg'
+LOOP_COUNT_FOR_THRESHOLD_TO_REMOVE_IFG = 'loop_count_for_threshold_to_remove_ifg'
+PHASE_UNWRAP_ERROR_THRESHOLD = 'phase_unwrap_error_threshold'
+MAX_LOOP_LENGTH = 'max_loop_length'
+SUBTRACT_MEDIAN_IN_CLOSURE_CHECK = 'subtract_median_in_closure_check'
+MAX_LOOP_COUNT_FOR_EACH_IFGS = 'max_loop_count_for_each_ifgs'
```

Comment:
Is it possible to shorten any of the param names? If properly documented in code and config file it won't matter that the name itself is not as descriptive

Reply:
addressed in https://github.com/GeoscienceAustralia/PyRate/pull/307/commits/8a056a5c06076db0fcbc5dc2c9d18dbdaad1dee1